meson.build

```diff
@@ -33,8 +35,9 @@ int main(int argc, char **argv)
 if not cxx.links(code, dependencies: deps, name: 'libicecc links dynamically')
     icecc = dependency('icecc', static: true)
     liblzo2 = cxx.find_library('lzo2')
-    libcapng = dependency('libcap-ng')
-    deps += [libcapng, liblzo2]
+    libcapng = dependency('libcap-ng', required: false)
+    libzstd = cxx.find_library('zstd', required: false)
```

Comment:
On my work machine I needed to link against `zstd` because it was required by `libicecc.a`. Was that added to `icecc` recently?

Reply:
Yes, it was added in the last release